### ISSUE TITLE
Update new bed calculation numbers to chart

### ISSUE THREAD

**Issue by hnykda** [epidemics/covid#296]
FE part for #264 , ideally solving #261 too.

AC:
* bed line is correcly showing up in charts

**Comment by mathijshenquet** [comment]
I propose that we just expose beds_per_100k for this. Then the frontend can do the necissary calculations.

**Comment by hnykda** [comment]
OK. So do you mean:

1. taking source data from https://github.com/epidemics/epifor/pull/15/
2. adding them to https://github.com/epidemics/epimodel-covid-data/
3. exposing them as https://github.com/epidemics/epimodel/pull/29/files

?

**Comment by mathijshenquet** [comment]
Yes except that the necassary data is already in `epimodel-data/hopsital-capacity.csv`. I think they used `rates.csv` to compute the capacity columns (right @jez4 ?)

**Comment by jez4** [comment]
@mathijshenquet I took the data from @veselatereza .. I have added only codes instead of country names.

**Comment by hnykda** [comment]
OK, cool, that's IMO correct and therefore steps 1. and 2. are not necessary.

**Comment by hnykda** [comment]
same as #296
